peer/brontide.go

```diff
@@ -49,6 +50,12 @@ const (
 	// pingInterval is the interval at which ping messages are sent.
 	pingInterval = 1 * time.Minute
 
+	// pingTimeout is the amount of time we will wait for a pong response
+	// before considering the peer to be unresponsive.
+	//
+	// This MUST be a smaller value than the pingInterval.
+	pingTimeout = 30 * time.Second
```

Comment:
Derivation here was just `interval/2`? I wonder if we should try to be more aggressive here...

Reply:
If we are too aggressive we will get in disconnect loops with our peers which I think would be a far more upsetting outcome. Tor can be very slow at times but even with Tor I'd imagine we can get away with something more aggressive. How aggressive? I'm not totally sure.